contracts/OpenSTUtility.sol

```diff
@@ -574,31 +576,6 @@ contract OpenSTUtility is Hasher, OpsManaged, STPrimeConfig {
         return uuids.length;
     }
 
-    /*
-     *  Registrar functions
-     */
-    /// @dev for v0.9.1 tracking Ethereum mainnet on the utility chain
-    ///      is not a required feature yet, so the core is simplified
-    ///      to uint256 valueChainId as storage on construction
-    // function addCore(
```

Comment:
why is `addCore` removed?

Reply:
it was dead-code until v0.9.3; so now either this function should not be removed or the constructor needs to have a setter for the core contract that represents OpenSTValue